ButteryToast/Toast.swift

```diff
@@ -91,20 +98,31 @@ open class Toast: Operation {
     topVC.view.addSubview(self.view)
     
     let yOffset: CGFloat
-    if #available(iOS 11.0, *) {
-      yOffset = topVC.view.safeAreaInsets.top
-    } else if topVC.navigationController != nil {
-      yOffset = topVC.topLayoutGuide.length
-    } else {
-      yOffset = UIApplication.shared.statusBarFrame.size.height
+    switch orientation {
+    case .top:
+      if #available(iOS 11.0, *) {
+        yOffset = topVC.view.safeAreaInsets.top
+      } else if topVC.navigationController != nil {
+        yOffset = topVC.topLayoutGuide.length
+      } else {
+        yOffset = UIApplication.shared.statusBarFrame.size.height
+      }
+      self.view.topAnchor.constraint(equalTo: topVC.view.topAnchor, constant: yOffset).isActive = true
+    case .bottom:
+      if #available(iOS 11.0, *) {
+        yOffset = -topVC.view.safeAreaInsets.bottom
+      } else if topVC.navigationController != nil {
```

Comment:
this check was specific to handling status bar so isn't relevant to bottom I think. This actually breaks support for presenting on bottom if a tab bar is present.